alibi_detect/saving/_pytorch/loading.py

```diff
@@ -18,6 +18,7 @@
 
 def load_model(filepath: Union[str, os.PathLike],
```

Comment:
Extra functionality sneaking into this PR... Worth adding changelog entries to this PR so everything is documented and not missed upon release?

Reply:
Not 100% clear what you mean here? The functionality of passing kwargs to `load_detector`? It is extra functionality but is interlinked with the PR, as `custom_objects` needs to be passed to `load_detector`.

Reply:
No need, but would appreciate a changelog as part of the PR.